oshmem/mca/spml/ucx/spml_ucx.h

```diff
@@ -280,6 +294,12 @@ extern int mca_spml_ucx_team_fcollect(shmem_team_t team, void
 extern int mca_spml_ucx_team_reduce(shmem_team_t team, void
         *dest, const void *source, size_t nreduce, int operation, int datatype);
 
+extern int mca_spml_ucx_rkey_store_create(mca_spml_ucx_rkey_store_t *store,
+                                          ucp_context_h ucp_context, int size);
+extern void mca_spml_ucx_rkey_store_destroy(mca_spml_ucx_rkey_store_t *store);
+extern ucp_rkey_h mca_spml_ucx_rkey_store_get(mca_spml_ucx_rkey_store_t *store, const ucp_rkey_h target);
+extern void mca_spml_ucx_rkey_store_put(mca_spml_ucx_rkey_store_t *store, const ucp_rkey_h target);
+
```

Comment:
i'd remove it and make these funcs static, as they are only used in a single file

Reply:
kept only first two which are needed for init